docs/current/cli/752479-guides.md

```diff
@@ -0,0 +1,7 @@
+---
+slug: /cli/guides
+---
+
+# Guides
+
+- [Proxy Requests to the Dagger API](./guides/632021-proxy-requests-api.md)
```

Comment:
I know we discussed having both these guides at the root of the category, but I was wondering if we should move this into the Guides section. This is just guesswork but I think that "using a shell script" will be a much more common use case for the CLI than "proxying requests". WDYT?